database/migrations/20240915183745-create-vehicles-table.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Vehicles', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      license: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        unique: true
+      },
+      brand: {
+        type: Sequelize.STRING,
+        allowNull: true
+      },
+      tradeName: {
+        type: Sequelize.STRING,
+        allowNull: true
+      },
+      dateFirstAllowed: {
+        type: Sequelize.DATE,
+        allowNull: true
+      },
+      dateFirstRegistration: {
+        type: Sequelize.DATE,
+        allowNull: true
+      },
+      price: {
+        type: Sequelize.INTEGER,
+        allowNull: true
+      },
+      vehicleType: {
+        type: Sequelize.STRING,
+        allowNull: true
+      },
+      interiorType: {
+        type: Sequelize.STRING,
+        allowNull: true
+      },
+      color: {
+        type: Sequelize.STRING,
+        allowNull: true
+      },
+      totalHorsepower: {
+        type: Sequelize.STRING,
+        allowNull: true
+      },
+      primaryFuelType: {
+        type: Sequelize.STRING,
+        allowNull: true
+      },
+      secondaryFuelType: {
+        type: Sequelize.STRING,
+        allowNull: true
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+
```

Comment:
Are we going to create script to migrate all past sightings?

Reply:
I think so yes. Would you want that to be a part of this PR? We can also just take the prod database and run the script locally

Reply:
Just create a separate PR for the script i guess